File: c/12e.c
#include <stdio.h>
#include <string.h>
int main(){
    // int input
    int input;
    scanf("%d", &input);
    for (int i = 0; i < 13; i++) {
        printf("%d * %d = %d\n", input, i, input * i);
    }
}

File: js/12e.js
//12e 10/10 program 
// imports
const prompt = require("prompt-sync")({ sigint: true });
// get the input
const num = prompt();
// loop
for (let i = 0; i < 13 ; i++) {
    console.log(`${num} x ${i} = ${num*i}`);
}